crates/ui/src/input/state.rs

```diff
@@ -364,15 +365,12 @@ impl InputState {
 
         let text_style = window.text_style();
         let mouse_context_menu = MouseContextMenu::new(cx.entity(), window, cx);
+        let font_size = text_style.font_size.to_pixels(window.rem_size());
```

Comment:
Here is not correct, I am sure.

The font_size should use a relative value to store in state.

We only can use `.to_pixels(window.rem_size());` in render stage.